### ISSUE TITLE
Accessibility issue / alternative to Chromelens

### ISSUE THREAD

**Issue by XhmikosR** [GoogleChrome/lighthouse#1705]
I'm hitting a specific issue with accessibility for a test project I have: <https://xhmikosr.github.io/greek-vat-calculator/>.

I keep getting that `#resetBtn` has insufficient contrast. So I followed the help link to see the suggestions. I installed Chromelens, but I'm hitting <https://github.com/chromelens/chromelens/issues/18>.

So there are 2 things:

1. I even tried using `background-color: #a70e07; color: #fff` which results in [7.8 contrast ratio](http://leaverou.github.io/contrast-ratio/#%23fff-on-%23a70e07) yet I still get that the audit failed. So it seems there's a bug somewhere.
2. Is there any alternative to Chromelens? If so, it should be mentioned in the docs since Chromelens doesn't seem to work properly

**Comment by XhmikosR** [comment]
This is still a valid issue. I'm not sure if it's a bug with axe or something else...

**Comment by patrickhulce** [comment]
related #2214

**Comment by ngzhian** [comment]
Hi @XhmikosR , thanks for reporting this. This is now fixed in [0.0.7](https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd), please update the extension. I have just tried it on your website, https://xhmikosr.github.io/greek-vat-calculator/, and the issue is not longer there. Let me know if this is still a problem, thanks!

**Comment by XhmikosR** [comment]
@ngzhian: thanks! Now it indeed works.
